### ISSUE TITLE
Add option to plot histogram / density plot

### ISSUE THREAD

**Issue by el-meyer** [el-meyer/airship#49]
Port over from commented out section in Boxplot

**Comment by majkamichal** [comment]
How this feature should look like?

One should have a new section "Histogram" or "Univariate distribution" or something like this, where one has bunch of widgets that control a histogram of continuous variables?

**Comment by el-meyer** [comment]
@majkamichal, yes exactly. Then allow users to add color, fill, facet, etc., based on other variables. Maybe have a toggle switch to select between histogram and density plot.
